packages/cli/src/commands/source/catalog/add.ts

```diff
@@ -0,0 +1,160 @@
+import {UploadBatchCallbackData, CatalogSource} from '@coveo/push-api-client';
+import {Command, Flags, CliUx} from '@oclif/core';
+import {green} from 'chalk';
+import {
+  HasNecessaryCoveoPrivileges,
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {
+  readOrganizationPrivilege,
+  writeFieldsPrivilege,
+  writeSourceContentPrivilege,
+} from '../../../lib/decorators/preconditions/platformPrivilege';
+import {Trackable} from '../../../lib/decorators/preconditions/trackable';
+import {
+  withFile,
+  withCreateMissingFields,
+  withFolder,
+  withMaxConcurrent,
+} from '../../../lib/flags/sourceCommonFlags';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+import {errorMessage, successMessage} from '../../../lib/push/userFeedback';
+import {getFileNames} from '../../../lib/utils/file';
+import {bold} from 'chalk';
+import dedent from 'ts-dedent';
+
+export default class SourceCatalogAdd extends Command {
+  public static description =
+    'Index a JSON document into a Coveo Catalog source. See https://docs.coveo.com/en/2956/coveo-for-commerce/index-commerce-catalog-content-with-the-stream-api for more information.';
```

Comment:
```suggestion
    'Index a JSON document into a Coveo Catalog source. See https://docs.coveo.com/en/2956 for more information.';
```
(everything after the ID is ignored by the docs site, so we can shorten this. It means losing some information, but I think it's worth it for brevity)